doc/autodocs/.cvsignore

```diff
@@ -1,3 +0,0 @@
-Makefile
```

Comment:
Ouch, we still have some `.cvsignore` files? I guess we should globally rename them to `.gitignore`...